WordPress/gutenberg
# i18n: Use the WordPress package and remove the built-in implementation

closes #5984

This PR updates Gutenberg to use the npm published i18n package. 
It removes the built-in implementation of the package and all the related tools (babel-plugin, pot-to-php).

**Testing instructions**

 - Check that Gutenberg loads
 - Try another language, notice Gutenberg is translated properly.
 - Try running `npm run package-plugin` notice the pot and the php files are included properly in the generated zip.
